lib/private/converter.rb

```diff
@@ -71,6 +67,7 @@ def _convert(value, type)
     sig { params(value: T.untyped, type: T.untyped).returns(T.untyped) }
     def _convert_simple(value, type)
       return nil if value.nil?
```

Comment:
I think we may want some special handling for `nil` conversion? like `""` may be allowed to be nil?